client/my-sites/marketing/buttons/options.jsx

```diff
@@ -117,20 +117,22 @@ class SharingButtonsOptions extends Component {
 		switch ( postType.name ) {
 			case 'index':
 				label = this.props.translate( 'Front Page, Archive Pages, and Search Results', {
-					context: 'jetpack',
+					context: 'Show like and sharing buttons on',
 				} );
 				break;
 			case 'post':
-				label = this.props.translate( 'Posts' );
+				label = this.props.translate( 'Posts', { context: 'Show like and sharing buttons on' } );
 				break;
 			case 'page':
-				label = this.props.translate( 'Pages' );
+				label = this.props.translate( 'Pages', { context: 'Show like and sharing buttons on' } );
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **147** times:
`translate( 'Pages', { context: 'post type general name'} )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/post-type-context)